src/components/preview/index.js

```diff
@@ -0,0 +1,158 @@
+import anime from "animejs";
```

Comment:
Let's rename this file to `preview-animation.js`.
The index file doesn't seem to have to contain logic